pkg/util/util.go

```diff
@@ -33,3 +36,65 @@ func GetRequestHost(req *http.Request) string {
 	}
 	return host
 }
+
+// RefreshValue contains a composite cookie refresh value setting
+type RefreshValue struct {
+	RefreshDuration *time.Duration
+	RefreshPercent  uint8
+}
+
+// Set a value of a refresh
+func (r *RefreshValue) Set(s string) error {
```

Comment:
I mentioned in another spot, but adding it here to track in the right place:

I think this should be `func NewRefreshValue(s string) (*RefreshValue, error)`

Reply:
`Set` is the interface of the `pflag` `Value`: https://github.com/spf13/pflag/blob/master/flag.go#L187

Reply:
If that's the case, can you add a comment to the method explaining the rationale for the design?

But is that still the case? https://github.com/oauth2-proxy/oauth2-proxy/pull/946#discussion_r536748136

Based on the only usage I see and the note in that spot where it looked odd: https://github.com/oauth2-proxy/oauth2-proxy/pull/946/files#r536488965